Components/LandingPage.jsx

```diff
@@ -0,0 +1,84 @@
+import { View, Button, StyleSheet, Text, TouchableOpacity } from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+import React from 'react';
+import { Video } from 'expo-av';
+
+const LandingPage = ({ navigation }) => {
+  return (
+    <LinearGradient
+      colors={['#6a11cb', '#2575fc']}
+      start={{ x: 0, y: 0 }}
+      end={{ x: 1, y: 1 }}
+      style={styles.container}
+    >
+      <Video
+        source={require('./Homie.mp4')}
+        rate={1.0}
+        isMuted={true}
+        resizeMode="contain"
+        shouldPlay
+        isLooping
+        style={styles.logoVideo}
+      />
+      <Text style={styles.welcomeText}>Welcome to HomieHub</Text>
+      <Text style={styles.descriptionText}>
+        HomieHub is the ultimate tool to keep you and your roommates organized and connected.
+      </Text>
+      
+      <Text style={styles.memberText}>Already a member?</Text>
+      <TouchableOpacity style={styles.loginButton} onPress={() => navigation.navigate('Login')}>
+        <Text style={styles.buttonText}>Log In</Text>
+      </TouchableOpacity>
+      <TouchableOpacity style={styles.registerButton} onPress={() => navigation.navigate('Register')}>
+        <Text style={styles.buttonText}>Register</Text>
+      </TouchableOpacity>
+    </LinearGradient>
+  );
+};
+
+const styles = StyleSheet.create({
```

Comment:
Maybe we should store this css in a separate file? I'd assume we'll end up recycling some of it for other pages.